packages/expo-sensors/src/PlatformModule.ts

```diff
@@ -0,0 +1,27 @@
+import { EventEmitter } from 'expo-core';
+
+export default class PlatformModule {
+  emitter = new EventEmitter({} as any);
```

Comment:
Not sure we want this -- my understanding is that the Platform Module base class should implement start/stopObserving and addListener/removeListeners and then you can pass an instance of the Module to expo-core's EventEmitter constructor and it will set things up. The frontend of the module is responsible for creating the EventEmitter, not the backend.